README.md

```diff
@@ -8,18 +9,30 @@ The structure of this repository is heavily inspired by [vsund](https://github.c
 ## Table of Contents
 <!-- TOC GFM -->
 
-- [Dependencies](#dependencies)
-- [Installation](#installation)
+- [Dotbot Template](#dotbot-template)
+  - [Table of Contents](#table-of-contents)
```

Comment:
We don't need these.
```suggestion
```

Reply:
The rest will need to be de-dented after the removal of the above.

Reply:
Made changes as per requested.

Check [fb88d4f](https://github.com/ecarlson94/dotbot-template/pull/9/commits/fb88d4f5033a76023ff3644ad71c2b4bf7a6eafc)